RickAndMortyFunApp/NetworkServices/NetworkService.swift

```diff
@@ -6,56 +6,97 @@
 //  Copyright © 2019 Rusłan Chamski. All rights reserved.
 //
 
+import Alamofire
 import Foundation
 
 enum NetworkError: Error {
     case invalidURL
 }
 
+enum NetworkRouter: URLRequestConvertible {
+    static let baseURLString = "https://rickandmortyapi.com/api/character/"
+
+    case get(Int)
```

Comment:
This is getCharacters and not get. Later we'll add getLocations etc. Also, maybe better to add a name to associated value. So: `case getCharacters(page: Int)`